docs/usage/channels.rst

```diff
@@ -0,0 +1,452 @@
+.. currentmodule:: litestar.channels
+
+Channels
+========
+
+**Channels** are a group of related functionalities, built to facilitate the routing
+of event streams, which for example can be used to broadcast messages to WebSocket
+clients.
+
+Channels provide:
+
+1. Independent :term:`broker` backends, optionally handling inter-process communication
+   and data persistence on demand
```

Comment:
This looks like it's cut off mid sentence

Reply:
That's because it's not a full sentence. It's a continuation of the "Channels provide" frament, which is continued in the other list items. That's also why there should be no periods at the end. 

This applies to the rest of the document as well.